src/executor/source/utils/logging/Logger.ts

```diff
@@ -0,0 +1,66 @@
+import winston from 'winston'
+import DailyRotateFile from 'winston-daily-rotate-file'
+
+const levels = {
+  error: 0,
+  warn: 1,
+  info: 2,
+  http: 3,
+  debug: 4,
+}
```

Comment:
Could we use `enum` here from Typescript?

Reply:
Unfortunately not. The type required by winston can be found [here](https://github.com/winstonjs/winston/blob/master/lib/winston/config/index.d.ts) on lines 7-9